dmutils/api_stubs.py

```diff
@@ -213,3 +215,87 @@ def supplier(id=1234, contact_id=4321, other_company_registration_number=0, comp
         data['suppliers']['registrationCountry'] = 'country:NZ'
 
     return data
+
+
+def supplier_framework(
+    agreed_variations=True,
+    supplier_id=1234,
+    framework_slug='g-cloud-7',
```

Comment:
Is there a big issue with using a more recent framework as the defaults (e.g. 10, and prefill from 9)?

Reply:
I was just trying to keep the stubs in sync. default for the framework stub is g7.